humble/Dockerfile

```diff
@@ -0,0 +1,55 @@
+#  Copyright (C) 2018-2022 LEIDOS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+ARG DOCKER_ORG=usdotfhwastoldev
+ARG DOCKER_TAG=develop-humble
+FROM ${DOCKER_ORG}/carma-base:${DOCKER_TAG} as base_image
+
+FROM base_image as source-code
+ARG GIT_BRANCH=develop
+
+# Install astuff ros2 ssc_pm using tokens as arguments
+ARG TOKEN="NULL"
+
+# ROS2 checkout deps
+RUN mkdir -p ~/workspace_ros2/src
+
+# ROS2 checkout deps
+COPY --chown=carma . /home/carma/workspace_ros2/src/
+RUN chmod -R u+x ~/workspace_ros2/src/docker/
+RUN ~/workspace_ros2/src/docker/checkout.bash -ros2 -b ${GIT_BRANCH}
+
+#Build ros2 pkgs
+RUN ~/workspace_ros2/src/docker/install.sh -ros2 ${TOKEN}
+
+FROM base_image
+
+ARG BUILD_DATE="NULL"
+ARG VERSION="NULL"
+ARG VCS_REF="NULL"
+
+LABEL org.label-schema.schema-version="1.0"
+LABEL org.label-schema.name="carma-ssc-interface-wrapper-driver"
+LABEL org.label-schema.description="ssc interface wrapper driver for the CARMA Platform"
+LABEL org.label-schema.vendor="Leidos"
+LABEL org.label-schema.version=${VERSION}
+LABEL org.label-schema.url="https://highways.dot.gov/research/research-programs/operations/CARMA"
+LABEL org.label-schema.vcs-url="https://github.com/usdot-fhwa-stol/carma-ssc-interface-wrapper/"
+LABEL org.label-schema.vcs-ref=${VCS_REF}
+LABEL org.label-schema.build-date=${BUILD_DATE}
+
+COPY --from=source-code --chown=carma /opt/carma /opt/carma
+COPY --from=source-code --chown=carma /opt/ros/humble /opt/ros/humble
```

Comment:
Can we add comments for why we are copying over these two? "/opt/carma" to copy for over installed directories
 and /opt/ros/humble to copy over the apt installed packages

Reply:
Added the comments